caikit/core/data_model/streams/data_stream.py

```diff
@@ -567,6 +597,62 @@ def _from_jsonl_collection_generator(cls, dirname):
         for data_item in DataStream.chain(data_stream_list).flatten():
             yield data_item
 
+    @classmethod
+    def from_multipart_file(cls, filename: str) -> "DataStream[JsonDictValue]":
+        """Loads up a DataStream from a multipart file
+
+            The data items returned in the data stream are determined by the
```

Comment:
NIT: Funny indentation? I guess this is what the other docstrings in this file look like which is weird

Reply:
yeah I went for consistency here but could clean this up to make the PR a little bigger 😄